### DISCUSSION TITLE
Unable to find documentation on VL53L4CX tuning parameters.

### DISCUSSION THREAD

**Discussion by gramcracker** [stm32duino/Arduino_Core_STM32#2341] [Libraries]
I am trying to get the VL53L4CX to pick up the distance of an object 6 meters away, but the signal is too low. I would like to change the settings in the driver, but there close to 0 documentation on it.
If possible I would also like to alter the field of view, since I only need to get the distance of a single object near the center.

**Comment by fpistm** [comment]
Hi @gramcracker ,
@cparata could you help on this subject, please?

**Comment by cparata** [comment]
Hello @gramcracker ,
I asked to our ToF experts. I hope to come back to you with a sequence of commands that you can call in your application to reach your goal.
Best Regards,
Carlo

**Comment by cparata** [comment]
Hello @gramcracker ,
I talked with ST ToF experts. Basically if you open the datasheet that you can find [here](https://www.st.com/resource/en/datasheet/vl53l4cx.pdf) at chapter 5.3 you can see a table where it is clear that 6 meters as target measurement is very tough to reach. You can reach this measurement only in a dark room with a target object with high reflectance (like a white wall) and with a full field view (reducing the ROI size you will decrease also the amount of the signal). So, in normal light condition of an indoor room it's practically impossible to get a good distance estimation for a target at 6 meters and typically working conditions are with target with a distance up to maximum 5 meters.
Best Regards,
Carlo
